app/models/service.rb

```diff
@@ -11,6 +11,7 @@ class Service < ApplicationRecord
   MAIRIE = "Mairie"
 
   # Relations
+  belongs_to :territory, optional: true
   has_many :agents, dependent: :nullify
   has_many :motifs, dependent: :destroy
```

Comment:
Je pense que pour être plus sereins sur cette migration on peut passer à cette config :

```suggestion
  has_many :agents, dependent: :restrict_with_exception
  has_many :motifs, dependent: :restrict_with_exception
```